src/components/TrapFocus/tests/TrapFocus.test.tsx

```diff
@@ -96,49 +96,11 @@ describe('<TrapFocus />', () => {
       .find('button')
       .getDOMNode();
 
-    const trapFocus = mountWithAppProvider(
-      <TrapFocus>
-        <TextField label="" value="" onChange={noop} autoFocus />
-        <TextField label="" value="" onChange={noop} autoFocus />
-      </TrapFocus>,
-    );
-
     const event: FocusEvent = new FocusEvent('focusout', {
       relatedTarget: externalDomNode,
     });
     Object.assign(event, {preventDefault: jest.fn()});
 
-    describe('prevents default when focus moves to an external node', () => {
```

Comment:
We'll no longer handle focus when `relatedTarget` is null. Instead, a later PR will handle keyboard inputs such as <kbd>tab</kbd> and focusing elements manually.

Reply:
Was this test failing? Should we remove it here or in your next PR?

Reply:
It was failing because we no longer run anything past the early return:

https://github.com/Shopify/polaris-react/blob/5731ce8f5e09960100010677c00d17e0407e8395/src/components/TrapFocus/TrapFocus.tsx#L76-L78

Without the early return, if `relatedTarget` was null we would still manage and set focus. So we should return it here.